dash_docs/chapters/dash_enterprise/dash_enterprise_chapters.py

```diff
@@ -3241,98 +3387,304 @@ def snapshot_page(n_clicks, url, wait_selector):
+
+    rc.Markdown(
+        '''
+        ### Please make sure you have the correct access rights and the repository exists
+        '''
+    ),
+
+    rc.Markdown(
+        '''
+        ```shell
+        $ git push plotly master
+        [...]
+        dokku@<dash-enterprise-server>: Permission denied (publickey).
+
+        fatal: Could not read from remote repository.
+
+        Please make sure you have the correct access rights
+
+        and the repository exists.
+        ```
+        '''
+    ),
+
+    rc.Markdown(
+        '''
+        This can happen when your `~/.ssh/config` is improproperly configured. See the Modify SSH Config section on the <dccLink href="/dash-enterprise/ssh" children="ssh"/> page.
+        page for more details. Watch out for trailing slashes in your Dash Enterprise Server domain!
+        '''
+    ),
+
+    rc.Markdown(
+        '''
+        ### remote: Access denied
+        '''
+    ),
+
+    rc.Markdown(
+        '''
+        ```shell
+        $ git push plotly master
+        [...]
+        Counting objects: 3, done.
+        Delta compression using up to 8 threads.
+        Compressing objects: 100% (3/3), done.
+        Writing objects: 100% (3/3), 5.18 KiB | 2.59 MiB/s, done.
+        Total 3 (delta 0), reused 0 (delta 0)
+
+        remote: User <your-user-name> does not have permissions to run git-hook on <your-dash-app>, or <your-dash-app> does not exist
+        remote: Access denied
+        To <your-dash-enterprise-host>/GIT/<your-dash-app>
+        ! [remote rejected] master -> master (pre-receive hook declined)
+
+        error: failed to push some refs to '<your-dash-enterprise-host>:<your-dash-app>'
+        '''
+    ),
+
+    rc.Markdown(
+        '''
+        ```shell
+        $ git push plotly master
+        [...]
+        ssh: connect to host <your-dash-enterprise-host> port 22: Connection timed out
+
+        fatal: Could not read from remote repository.
+
+        Please make sure you have the correct access rights
+
+        and the repository exists.
+        ```
+        '''
+    ),
+
+    rc.Markdown(
+        '''
+        If deploying with SSH, then this can happen when your `~/.ssh/config` is improperly configured. See  <dccLink href="/dash-enterprise/ssh" children="ssh"/> for more details.
+        
+        If deploying with HTTPS, then this can happen if your username or password is incorrect. Note that usernames and passwords in Dash Enterprise are case sensitive; e.g. the username `Chris.Parmer` is different than the username `chris.parmer`.
+        Git can sometimes cache these usernames after you've typed them. Run `$ git config credential.helper` to verify and correct the username. 
+        Also, some operating system's password managers can cache usernames as well. Open your operating system's password manager and verify the username & password if you are deploying with `https`, running into access denied errors, and `git` isn't asking you for your username and password.
```

Comment:
![image](https://user-images.githubusercontent.com/1280389/100397066-e0450c00-3015-11eb-98a5-4637861f3d97.png)


```suggestion
        If deploying with HTTPS, then this can happen if your username or password is incorrect. Note that usernames and passwords in Dash Enterprise are case sensitive; e.g. the username `Chris.Parmer` is different than the username `chris.parmer`.
        
        Git can sometimes cache these usernames after you've typed them. Run `$ git config credential.helper` to verify and correct the username. 
        
        Also, some operating system's password managers can cache usernames as well. Open your operating system's password manager and verify the username & password if you are deploying with `https`, running into access denied errors, and `git` isn't asking you for your username and password.
```